app/reducers/trezor.js

```diff
@@ -120,8 +125,6 @@ export default function trezor(state = {}, action) {
     case TRZ_WALLET_CLOSED:
       return {
         ...state,
-        device: null,
-        deviceLabel: "",
```

Comment:
This allows the same wallet to be closed and opened again without having to disconnect the trezor once.